lib/csv_parser_client.rb

```diff
@@ -9,6 +9,14 @@ def self.get_tube_rack_scan_results(tube_rack_barcode, object_to_add_errors_to)
     endpoint = configatron.tube_rack_scans_microservice_url
     response = Net::HTTP.get_response(URI(endpoint + tube_rack_barcode))
 
+    if response.body.nil?
+      error_message =
+        "Scan could not be retrieved for tube rack with barcode #{tube_rack_barcode}. Service responded with status code #{response.code}"
+      error_message += " and message #{response.message}."
```

Comment:
Not a big deal at all, but if three lines are needed anyway (presumably to satisfy the linters) it might look neater if you split that long first string into two parts at the break between the sentences.  Then you won't have a short line with just the variable name and a really long one following it.